reference/configuration/framework.rst

```diff
@@ -196,9 +197,35 @@ see :doc:`/cookbook/request/load_balancer_reverse_proxy`.
 form
 ~~~~
 
+.. _form-enabled:
+
+enabled
+.......
+
+**type**: ``boolean`` **default**: ``false``
+
+Whether or not to enable support for the Form component.
+
 csrf_protection
 ~~~~~~~~~~~~~~~
 
+.. _csrf-protection-enabled:
+
+enabled
+.......
+
+**type**: ``boolean`` **default**: ``true`` if form support is enabled, ``false``
+otherwise
+
+This option can be used to disable CSRF protection of forms.
```

Comment:
Here we could also add some broader explanation: *"For security reasons you should never disable CSRF protection. However, in these scenarios it may be useful to disable it: lorem ipsum, dolor sit amet. etc."*